build-system/tasks/visual-diff.js

```diff
@@ -455,6 +455,19 @@ async function snapshotWebpages(percy, page, webpages, config) {
     await verifyCssElements(page, url, webpage.forbidden_css,
         webpage.loading_incomplete_css, webpage.loading_complete_css);
 
+    if (webpage.loading_complete_delay_ms) {
+      if (typeof webpage.loading_complete_delay_ms !== 'number' &&
+          webpage.loading_complete_delay_ms > 0) {
+        log('verbose', 'Waiting',
+            colors.cyan(webpage.loading_complete_delay_ms + 'ms'),
+            'for loading to complete');
+        await sleep(webpage.loading_complete_delay_ms || 0);
+      } else {
+        log('warning', 'Skipping unknown delay',
+            webpage.loading_complete_delay_ms);
+      }
+    }
+
```

Comment:
If we do it before `verifyCssElements`, it won't have the same effect.  We need to wait at least `loading_complete_delay_ms` milliseconds from the moment the CSS class is applied in order to wait for the transition to play out.  If we do it beforehand, it's possible that we will wait less than that amount of time, which could flake the tests, since the transition could be in progress.